vlib/benchmark/README.md

```diff
@@ -1,6 +1,7 @@
 ## Description:
 
-`benchmark` provides an easy way to measure how fast a piece of code is and to produce a readable report about it.
+`benchmark` provides an easy way to measure how fast a piece of code is and
+to produce a readable report about it.
```

Comment:
Remove `to` here.  It's... awkward phrasing with it, but perfectly valid without.

Reply:
Debatable... I'd like to reword it completely. I'll give it another shot.